pkg/network/infraconfigurators/masquerade_test.go

```diff
@@ -464,8 +497,60 @@ func mockIPTablesBackendAllPorts(handler *netdriver.MockNetworkHandler, proto ip
 		vmIP).Return(nil)
 }
 
+func mockIstioNetfilterCalls(handler *netdriver.MockNetworkHandler, proto iptables.Protocol, nftIPString string, vmIP string, gwIP string) {
+	for _, chain := range []string{"output", "KUBEVIRT_POSTINBOUND"} {
+		handler.EXPECT().NftablesAppendRule(proto, "nat",
+			chain, "tcp", "dport", fmt.Sprintf("{ %s }", strings.Join(istio.ReservedPorts(), ", ")),
+			nftIPString, "saddr", GetLoopbackAdrress(proto), "counter", "return").Return(nil)
+	}
+
+	podIP := netlink.Addr{IPNet: &net.IPNet{IP: net.ParseIP("10.35.0.2"), Mask: net.CIDRMask(24, 32)}}
+	srcAddressesToSnat := getSrcAddressesToSNAT(proto)
+	dstAddressesToDnat := getDstAddressesToDNAT(proto, podIP)
+	if proto == iptables.ProtocolIPv4 {
+		handler.EXPECT().ReadIPAddressesFromLink("eth0").Return(podIP.IP.String(), "", nil)
+	}
+	handler.EXPECT().NftablesAppendRule(proto, "nat",
+		"KUBEVIRT_POSTINBOUND", nftIPString, "saddr", fmt.Sprintf("{ %s }", strings.Join(srcAddressesToSnat, ", ")),
+		"counter", "snat", "to", gwIP).Return(nil)
+	handler.EXPECT().NftablesAppendRule(proto, "nat",
+		"output", nftIPString, "daddr", fmt.Sprintf("{ %s }", strings.Join(dstAddressesToDnat, ", ")),
+		"counter", "dnat", "to", vmIP).Return(nil)
+	handler.EXPECT().NftablesAppendRule(proto, "nat",
+		"KUBEVIRT_PREINBOUND",
+		"counter", "dnat", "to", vmIP).Return(nil).Times(0)
+}
+
+func getSrcAddressesToSNAT(proto iptables.Protocol) []string {
+	srcAddressesToSnat := []string{GetLoopbackAdrress(proto)}
+	if proto == iptables.ProtocolIPv4 {
+		srcAddressesToSnat = append(srcAddressesToSnat, istio.GetLoopbackAddress())
+	}
+	return srcAddressesToSnat
+}
+
+func getDstAddressesToDNAT(proto iptables.Protocol, podIP netlink.Addr) []string {
+	dstAddressesToDnat := []string{GetLoopbackAdrress(proto)}
+	if proto == iptables.ProtocolIPv4 {
+		dstAddressesToDnat = append(dstAddressesToDnat, podIP.IP.String())
+	}
+	return dstAddressesToDnat
+}
+
 func getProtocols(optionalIPProtocol ...iptables.Protocol) []iptables.Protocol {
 	return append(
 		[]iptables.Protocol{iptables.ProtocolIPv4},
 		optionalIPProtocol...)
 }
+
+func getReservedPortList() []int {
```

Comment:
Same

Reply:
I don't know what this means right now. Please elaborate so I can resolve it.

Reply:
I think it was about the `get` prefix that is not needed.
Not going to insist on this nit.